netlify.toml

```
@@ -7,7 +7,14 @@
   to = "/howto/integration"
   status = 301
 
-# To keep old download links alive
+# The directory index is redirected to the new canonical location
+[[redirects]]
+  from = "/download"
+  to = "https://sources.nginx.org/unit"
```

Avoid double-redirection

```suggestion
  to = "https://sources.nginx.org/unit/"
```

---

Mind explaining this?